chia/types/blockchain_format/program.py

```diff
@@ -267,7 +267,7 @@ def run_as_generator(
         self, max_cost: int, flags: int, *args
     ) -> Tuple[Optional[int], Optional[SpendBundleConditions]]:
 
-        serialized_args = b""
+        serialized_args = bytearray()
```

Comment:
I imagine operating on `bytearray` is faster than on `bytes`, so this seems good.

Reply:
that's my understanding. in this case it's probably quite marginal